libcxx/include/__config

```diff
@@ -881,14 +898,25 @@ typedef __char32_t char32_t;
 
 // Inline namespaces are available in Clang/GCC/MSVC regardless of C++ dialect.
 // clang-format off
-#  define _LIBCPP_BEGIN_NAMESPACE_STD namespace _LIBCPP_TYPE_VISIBILITY_DEFAULT std {                                  \
+#  define _LIBCPP_BEGIN_NAMESPACE_STD _LIBCPP_DIAGNOSTIC_PUSH                                                          \
```

Comment:
I would suggest defining a shorthand for these to avoid cluttering the namespace definition too much.

Reply:
As discussed, this is optional.